### PR TITLE
New package: Samsung.EasyFileShare version 1.2.4

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111991&drop=dogfoodAlpha

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by stephengillie** [comment]
image

File: manifests/s/Samsung/EasyFileShare/1.2.4/Samsung.EasyFileShare.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.EasyFileShare
+PackageVersion: 1.2.4
+Installers:
```
Comment (stephengillie) [inline manifests/s/Samsung/EasyFileShare/1.2.4/Samsung.EasyFileShare.installer.yaml:6]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
Installers:
```

**Comment by stephengillie** [comment]
Manual Validation failed with:
> 2023-07-21 15:19:02.145 [CLI ] Installer [X64,exe,Unknown,en-US] not applicable: Architecture was excluded by caller : X64  
> 2023-07-21 15:19:02.146 [CLI ] No suitable installer found for manifest Microsoft.DotNet.Framework.DeveloperPack_4 with version 4.8.1  
> 2023-07-21 15:19:02.147 [CLI ] Terminating context: 0x8a150104 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DependenciesFlow.cpp:126  

It appears that the dependency doesn't have a valid matching installer.

**Comment by upintheairsheep** [comment]
> image

Tag: Dependencies

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

Unfortunately this is a Windows Feature, and I do not know when (if?) we will support this.

**Comment by Trenly** [comment]
[Policy] Windows Feature

**Pull request closed without merging**